src/domains/FeedbackPortal/ReportForm/ReportForm.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+
+import FormBase from '../FormBase';
+
+interface DefaultProps {
+    townhallId: string;
+}
+interface FormProps {
+    title: string;
+    mainDescription: string;
+    icon: JSX.Element;
+    reportType: 'Feedback' | 'Bug';
+    townhallId?: string;
+}
+
+export default function ReportForm({
+    title,
+    mainDescription,
+    icon,
+    reportType,
+    townhallId,
+}: FormProps & DefaultProps) {
+    return (
+        <Grid container spacing={5}>
+            <Grid item xs={12}>
+                <Typography variant='h4' align='center'>
+                    {title}
+                    {React.cloneElement(icon)}
```

Comment:
Out of curiosity, does just using `{icon}` not work here (it may have to be of type React.ReactNode or something like that)?

Reply:
Kind of weird but first time I try coding this without the React.cloneElement, I would get an error. But, now it seems like I can just render the icon by itself.  Thanks for catching this.